src/App.jsx

```diff
@@ -1,7 +1,7 @@
 import { useState } from 'react';
```

Comment:
Creating your `newTodo` inside `addTodo` in this component works just fine but since `addTodo` is mainly purposed with adding a todo to state it's better to have `handleAddTodo` in `TodoListItem` be in charge of creating a new todo. This better separates the concerns of your functions and components.